### PR TITLE
Personal portfolio

### PR DESCRIPTION
**Pull request by Hasinarahman**
Creating personal portfolio by using html,css and js.
Capture

Figma Design
  (https://www.figma.com/design/IULNZAus8DhpOTmHgeEpDu/Portfolio?t=AvdQd1u18VKBqbar-0)

### CODE REVIEW

**Comment by Hasinarahman** [comment]
Hi dear @Najeeba-Qarqin,
finally I finished my project please ones review and give me feedback.

**Comment by Najeeba-Qarqin** [comment]
Hi @Hasinarahman !
Well done you have done an amazing project👏

✔️ Created a professional Repo.
✔️ Correct use of Gitflow
✔️ Commit messages are meaningful 
✔️ Setup the linters correctly. 
✔️ No linter errors 
✔️ You have followed the Personal Portfolio project equirments
✔️ You have followed the all design of figma from UX/UI student
✔️ all of your links works good
✔️ Created the pull request correctly. 
✔️ Good and meaningful PR description 
✔️Your code is clean
✔️Your repo has a professional README.md
✔️Your code functionality is works fine. 
Just there is little missing in design of the project .

**Comment by Najeeba-Qarqin** [comment]
In mobile version the heading of home page and the paragraph in about page should be separate. 
Your design: 
IMG-20240804-WA0004


Figma design: 
IMG-20240804-WA0006

**Comment by Najeeba-Qarqin** [comment]
In the popup of your projects the languages section is not clear.
You should add them at the bottom of the module. 

IMG-20240804-WA0002

**Comment by Najeeba-Qarqin** [comment]
And also the project cards are three in the row it should be two as the Figma design. 

Your design:

IMG-20240804-WA0003


Figma design:


IMG-20240804-WA0005

**Comment by Hasinarahman** [comment]
Hi @Najeeba-Qarqin,
I emplement all of your sugestion so please review again.

**Comment by Najeeba-Qarqin** [comment]
@Hasinarahman good job your project doesn't have any issues now and the status of your project is approved you can mention the mentor.

**Comment by Hasinarahman** [comment]
Hi sir @Ahzamir
please check my project.

**Comment by Hasinarahman** [comment]
@ahzamir
sir please check my project.

File: styles.css
Comment (ahzamir) [inline styles.css:1]: - Well done @Hasinarahman  You implemented the design very well, and there is just 2 small changes needed on your project inorder to get the final approval:

### Humberger menu close button:
- The close button is not same as the Figma Design close button:
  - Your Design:
image
  - Figma Design:
  
image


### Pop-up Modal:
- The pop-up modal for your Desktop version is too small, it should be bigger then this:
image

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+## Technologies Used 🛠️
+- HTML
+- CSS
+- JS
+- Git
+- GitHub
+- Figma Design
+
+## Figma Design
+  (https://www.figma.com/design/IULNZAus8DhpOTmHgeEpDu/Portfolio?t=AvdQd1u18VKBqbar-0)
```
Comment (ahzamir) [inline README.md]: - Add it like this:
```
- [Figma Design](https://www.figma.com/design/IULNZAus8DhpOTmHgeEpDu/Portfolio?t=AvdQd1u18VKBqbar-0)
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+
+- Email: (hasinarahmani548@gmail.com)
+- linked in: (linkedin.com/in/hasina-rahmani-4a21a9311)
+- Github:
+- UI/UX Designers: 
+- 1 Maryam Yousufi: facebook (https://www.facebook.com/ada.aaramsh?mibextid=LQQJ4d), linkedin (https://www.linkedin.com/in/maryam-yousufi-)
+- 2 Sara Shirzad: linkedin (https://www.linkedin.com/in/sara-shirzad-478696267?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app)
+- 3 Sadaf Ahmadi: facebool (https://www.facebook.com/profile.php?id=100062037753963&mibextid=kFxxJD)
+- 4 Farkhunda Raufi:
+
```
Comment (ahzamir) [inline README.md]: - Same in here add the links properly like this:
```
- [Link Text](The Link)
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Hasinarahman 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Well done Hasina, you have done a great job by implementing 98% of the project's requirements, just some minor changes needed inorder to get the final approval please check the comments below:

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Personal Portfolio and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
❌Your design is same as the Figma Design

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Hasinarahman** [comment]
Hello sir @ahzamir 
I emplement all of your sugestion so please review again.

**Comment by Hasinarahman** [comment]
Hello sir @ahzamir 
please check my project.

**Comment by Hasinarahman** [comment]
hi @ahzamir 
sir I did all the feedback you give me and this is the third week soo please check my project.

**Review by ahzamir — APPROVED** [review]
### Hi @Hasinarahman 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the JS Capstone project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
✔️Your design is the same as the Figma designn

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Hasinarahman** [comment]
Thanks sir Ok I will share
@ahzamir

**Pull request merged**